make/kube

```diff
@@ -6,24 +6,24 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 cd "${GIT_ROOT}"
 source .envrc
 
-CREATE_HELM_CHART=false
-BUILD_TARGET=kube
-
 if [ "${1:-}" = "helm" ]; then
-    CREATE_HELM_CHART=true
     BUILD_TARGET=helm
-fi
-
-if [ "${BUILD_TARGET}" = "kube" ]; then
+else
+    BUILD_TARGET=kube
     # Overrides when generating kube config files instead of helm charts.
     FISSILE_OUTPUT_DIR="${PWD}/kube"
     ENV_FILES="$(echo bin/settings/*.env bin/settings/kube/*.env)"
+    DOMAIN=${DOMAIN:-$(grep '^DOMAIN=' bin/settings/network.env | sed 's/DOMAIN=//')}
```

Comment:
Ah.. since `cd $GIT_ROOT` is at the beginning of this script, I think relative references are OK.